src/runtime/containerd-shim-v2/metrics.go

```diff
@@ -8,7 +8,7 @@ package containerdshim
 import (
 	"context"
 
-	"github.com/containerd/cgroups"
+	"github.com/containerd/cgroups/stats/v1"
```

Comment:
How about:

```suggestion
	cgroupsv1 "github.com/containerd/cgroups/stats/v1"
```

That makes it clear which "thing" is at version 1.